django_redshift_backend/base.py

```diff
@@ -165,9 +164,10 @@ def create_model(self, model):
                 definition += " %s" % col_type_suffix
             params.extend(extra_params)
             # FK
-            if field.rel and field.db_constraint:
-                to_table = field.rel.to._meta.db_table
-                to_column = field.rel.to._meta.get_field(field.rel.field_name).column
+            print(dir(field))
+            if field.remote_field and field.db_constraint:
```

Comment:
`Field.rel` is removed. Use `remote_field`.

https://docs.djangoproject.com/en/2.0/releases/1.9/#field-rel-changes

Reply:
Oh... print func